src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.page_body {
+  margin: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Since `.page_body` seems to be a class for the `<body>` tag, it's unnecessary to create a class for it. The styles can be applied directly to the `body` tag.